content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -1033,6 +1033,18 @@ and legacy in-tree cloud providers), it sets this annotation on the Node to deno
 set from the command line flag (`--node-ip`). This IP is verified with the cloud provider as valid
 by the cloud-controller-manager.
 
+### volume.alpha.kubernetes.io/node-affinity (alpha) {#alpha-kubernetes-io-provided-node-ip}
+
+Type: Annotation
+
+Example: `volume.alpha.kubernetes.io/node-affinity: "constrain-with-nodes"`
+
+Used on: Node
+
+Node affinity and [nodeSelector](https://kubernetes.io/docs/reference/labels-annotations-taints/#schedulerkubernetesnode-selector) are conceptually similar, both enabling you to restrict the scheduling of your Pod to specific nodes in the cluster, based on node labels.
+
+See [node affinity](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity) for more information. 
```

Comment:
```suggestion
See [node affinity](/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity) for more information. 
```

Reply:
Same with this `/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity`, when I add this with the node affinity it's redirecting me to a `page not found`.
So may I add this address to the `node affinity` `https://github.com/kubernetes/website/blob/main/content/en/docs/concepts/scheduling-eviction/assign-pod-node.md/#node-affinity` ?